internal/pkg/client/cache/library.go

```diff
@@ -1,38 +1,53 @@
-// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
 // This software is licensed under a 3-clause BSD license. Please consult the
 // LICENSE.md file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
 package cache
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/sylabs/scs-library-client/client"
-	"github.com/sylabs/singularity/internal/pkg/sylog"
 )
 
 const (
 	// LibraryDir is the directory inside the cache.Dir where library images are cached
 	LibraryDir = "library"
 )
 
+/*
```

Comment:
please delete code, don't comment it out

Reply:
Agreed. Just something i missed before creating the PR.